reco_utils/dataset/criteo_dac.py

```diff
@@ -110,34 +117,64 @@ def _get_schema(include_label=True):
   return schema
 
 
-def _load_datafile(local_cache_path="dac.tar.gz", dbutils=None):
+def criteo_urls(size):
```

Comment:
i would drop this method in favor of storing the dict at the top of the file

Reply:
done in the next commit